packages/web-app-draw-io/src/App.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <iframe
+    id="drawio-editor"
+    ref="drawIoEditor"
+    :src="iframeSource"
+    :title="$gettext('Draw.io editor')"
+  />
```

Comment:
looks like this got lost over time ..... https://github.com/owncloud/web/pull/10702/files

Reply:
It has been removed intentionally: https://github.com/owncloud/web/pull/11175

I don't know the reasoning though, maybe @AlexAndBear can enlighten us?

Reply:
Ah - I remember the discussion ..... yeah. I am out of this. THX

Reply:
Sandboxing is not fully tested and breaks in some cases, for example a lot of collabora features did break.
We can enable sandbox later, but this must  go trough a lot of testing, so we don't break functionalities of the respective apps again.

Reply:
FYI: https://github.com/CollaboraOnline/online/issues/8921 ...

Reply:
Additionally, if this would be fixed: still more attributes must be enabled to make collabora work, e.g 
allow-modals, allow-forms (for epub export) and maybe more: We need to test it really well.  


<br class="Apple-interchange-newline">